binding/binding_test.go

```diff
@@ -278,6 +282,12 @@ func TestBindingFormForTime2(t *testing.T) {
 		"", "")
 }
 
+func TestFormBindingIgnoreField(t *testing.T) {
+	testFormBindingIgnoreField(t, "POST",
+		"/", "/",
+		"foo=bar", "")
```

Comment:
Your test is not fail without your changes in `binding/form_mapping.go`.
```suggestion
		"-=bar", "")
```

Reply:
That's right. I just force-pushed with this change. I run tests and they are still passing. 👍